### ISSUE TITLE
[Feature Request]Movie release year against the titles

### ISSUE THREAD

**Issue by onlytanmoy** [jurialmunkey/skin.arctic.horizon#230]
Please see below video.
The information page in AH skin does not show movie years against the movies.
Can the years of the movies be populated against movie title so like Deep Blue Sea 3 (2020)?
https://imgur.com/a/beXg5QJ

**Comment by rijsab** [comment]
Yes, would be great to have this directly next to title (similar to Aura)

**Comment by onlytanmoy** [comment]
Please have release year populated against the movie titles in the home screen widgets.
